simde/arm/neon/int8x16.h

```diff
@@ -116,6 +123,8 @@ simde_vmulq_s8(simde_int8x16_t a, simde_int8x16_t b) {
   simde_int8x16_t r;
 #if defined(SIMDE_NEON_NATIVE)
   r.n = vmulq_s8(a.n, b.n);
+#elif defined(SIMDE_NEON_WASM_SIMD128)
+  r.v128 = wasm_i8x16_mul(a.v128, b.v128);
```

Comment:
https://travis-ci.org/github/nemequ/simde/jobs/689963809#L569
``` make
../arm/neon/../../../simde/arm/neon/int8x16.h:127:12: error: implicit declaration of function 'wasm_i8x16_mul' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  r.v128 = wasm_i8x16_mul(a.v128, b.v128);
           ^
```

Reply:
That's really weird, [3 days ago](https://github.com/llvm/llvm-project/commit/3181273be73ac798c3d5b9081dd53d87a31c91b3), they removed that function. I'll fix once I get back on my pc.